lib/bolognese/author_utils.rb

```diff
@@ -36,9 +36,15 @@ def get_one_author(author)
             "schemeUri" => "https://orcid.org",
             "nameIdentifierScheme" => "ORCID" }.compact
         elsif ni["schemeURI"].present?
+          name_identifier = nil
+          if ni["__content__"].present?
+            name_identifier = ni["__content__"].to_s.strip
+            schemeURI = ni["schemeURI"].end_with?("/") ? ni["schemeURI"] : ni["schemeURI"] + "/"
```

Comment:
maybe abstract this out to its own method. it would make the code easier to read next time around.